src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -175,6 +192,34 @@ public double getGyroAngle() {
     }
   }
 
+  public DifferentialDriveKinematics getKinematics() {
+    return m_kinematics;
+  }
+
+  public Pose2d getPose() {
+    return m_odometry.getPoseMeters();
+  }
+
+  /**
+   * Sets the speed for the internal PID Controller on the Spark Max on the left
+   * side of the drivetrain
+   * 
+   * @param speed the speed to set in meters per second
+   */
+  public void setLeftSpeed(double speed) {
```

Comment:
`speed`->`velocity`

Our "informal" usage of speed generally means percent output, but velocity always means closed-loop meter/sec or deg/sec

Reply:
resolved in 93bb0d3